api_docs/construct-narrow.md

```diff
@@ -84,15 +90,17 @@ filters did.
 
 ### Message IDs
 
-The `near` and `id` operators, documented in the help center, use message
-IDs for their operands.
+The `with`, `near` and `id` operators, documented in the help center,
+use message IDs for their operands.
 
+* `with:12345`: Search for the conversation (channel and topic) which
+contains the message with ID `12345`.
```

Comment:
Where do we explain the precise semantics of the operator -- that it results in the query 

Also, this description doesn't explain what happens for a `with` operator in a DM conversation -- what happens then?

Reply:
I added them!